docroot/profiles/custom/cgov_site/themes/custom/cgov/package-lock.json

```diff
@@ -2545,6 +2503,14 @@
             "picomatch": "^2.0.4"
           }
         },
+        "axios": {
+          "version": "0.19.2",
+          "resolved": "https://registry.npmjs.org/axios/-/axios-0.19.2.tgz",
+          "integrity": "sha512-fjgm5MvRHLhx+osE2xoekY70AhARk3a6hkN+3Io1jc00jtquGvxYlKlsFUhmUET0V5te6CcZI7lcv2Ym61mjHA==",
+          "requires": {
+            "follow-redirects": "1.5.10"
+          }
+        },
```

Comment:
This surprises me.  Shouldn't this be axios 0.21.1 like the others?

This is from a list of required packages for the glossary app.  That also surprises me; when we switched to the generic app modules, I thought we would no longer need to reference those packages in the platform's build.